pkg/porter/build.go

```diff
@@ -188,19 +189,33 @@ func (p *Porter) preLint(ctx context.Context, file string) error {
 }
 
 func (p *Porter) getUsedMixins(ctx context.Context, m *manifest.Manifest) ([]mixin.Metadata, error) {
-	installedMixins, err := p.ListMixins(ctx)
+	ctx, span := tracing.StartSpan(ctx)
+	defer span.EndSpan()
+
+	usedMixins := make([]mixin.Metadata, len(m.Mixins))
+
+	g := new(errgroup.Group)
+	results := make(chan *mixin.Metadata, len(m.Mixins))
```

Comment:
Is there a particular reason why want to have a pointer here and then de-reference it later on line 207

Reply:
It was just a matter of where we wanted to dereference the pointer that was stored in that PackageMetadata interface. I've updated it to do less conversions.